### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by m-atia**

### CODE REVIEW

File: rules.txt [hunk 1]
```diff
@@ -855,11 +855,44 @@
...
-/services/crime-law-justice/inmates/send-mail-to-someone-in-jail-or-prison/ 301 /services/crime-law-justice/inmates/contact-an-inmate-by-mail/
-/services/crime-law-justice/inmates/find-an-inmate/ 301 /services/crime-law-justice/inmates/locate-an-inmate/
+/services/crime-law-justice/find-an-inmate/  301  /services/crime-law-justice/prisons-incarcerated-people-and-returning-citizens/finding-and-contacting-incarcerated-people/locate-an-incarcerated-person/
+/services/crime-law-justice/inmates/visit-a-person-in-jail-or-prison/  301  /services/crime-law-justice/prisons-incarcerated-people-and-returning-citizens/finding-and-contacting-incarcerated-people/visit-an-incarcerated-person/
+/services/crime-law-justice/inmates/send-mail-to-someone-in-jail-or-prison/ 301  /services/crime-law-justice/prisons-incarcerated-people-and-returning-citizens/finding-and-contacting-incarcerated-people/contact-an-incarcerated-person-by-mail/
+/services/crime-law-justice/inmates/find-an-inmate/ 301  /  services/crime-law-justice/prisons-incarcerated-people-and-returning-citizens/finding-and-contacting-incarcerated-people/locate-an-incarcerated-person/
```
Comment (karissademi) [inline rules.txt:861]: Looks like there's an extra space after the `301 / `

**Review by karissademi — CHANGES_REQUESTED** [review]
One tiny fix!

File: rules.txt (hunk 1 above)
Comment (karissademi) [reply rules.txt:861]: On line 861

Comment (m-atia) [reply rules.txt]: fix'd

**Pull request merged**